egs/fisher_callhome_spanish/s5/local/merge_lexicons.py

```diff
@@ -34,11 +34,12 @@
     if entries[0].lower() not in merged_lexicon:
         merged_lexicon.append(entries[0].lower())
 
-print("After adding the LDC data, the lexicon contains {} entries".format(len(merged_lexicon)))
+print "After adding the LDC data, the lexicon contains " \
+      + str(len(merged_lexicon)) + " entries."
 
 # Finally add the gigaword data
 gigaword = json.load(open(uw_gigaword))
-gigaword = reversed(sorted(gigaword.items(), key=operator.itemgetter(1)))
+gigaword = reversed(sorted(gigaword.iteritems(), key=operator.itemgetter(1)))
```

Comment:
you seem to have reversed the change of items to iteritems-- this breaks in python3.

Reply:
Thanks for pointing out. Its reverted back now.